<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>597</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Üye Kayıt</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Üye Sorgu</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Kitap Kayıt</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Kitap Sorgu</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Ödünç Verme</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Kitap ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Üye ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtOdncKitap">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtOdncUyeId">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOduncVer">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Odunc Ver</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Süre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtOdncSure">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kullanıcı Şifre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtUyeAd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtUyeSifre">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOturum">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Oturum Aç</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGiri">
    <property name="title">
     <string>Giriş</string>
    </property>
    <addaction name="actionUye_Kayit"/>
    <addaction name="action_ye_Sorgu"/>
   </widget>
   <widget class="QMenu" name="menuKitap">
    <property name="title">
     <string>Kitap</string>
    </property>
    <addaction name="actionKitap_Kayit"/>
    <addaction name="actionKitap_Sorgu"/>
   </widget>
   <widget class="QMenu" name="menu_d_n">
    <property name="title">
     <string>Ödünç</string>
    </property>
    <addaction name="actionOduncVer"/>
    <addaction name="actionOduncAl"/>
   </widget>
   <addaction name="menuGiri"/>
   <addaction name="menuKitap"/>
   <addaction name="menu_d_n"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUye_Kayit">
   <property name="text">
    <string>Üye Kayıt</string>
   </property>
  </action>
  <action name="action_ye_Sorgu">
   <property name="text">
    <string>Üye Sorgu</string>
   </property>
  </action>
  <action name="actionKitap_Kayit">
   <property name="text">
    <string>Kitap Kayıt</string>
   </property>
  </action>
  <action name="actionKitap_Sorgu">
   <property name="text">
    <string>Kitap Sorgu</string>
   </property>
  </action>
  <action name="actionOduncVer">
   <property name="text">
    <string>Ödünç Ver</string>
   </property>
  </action>
  <action name="actionOduncAl">
   <property name="text">
    <string>Ödünç Al</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
